src/base/memory_monitor_common.h

```diff
@@ -27,9 +27,18 @@
 #include <cstdint>
 #include <vector>
 
+constexpr char module_names[MMON_MODULE_LAST][DB_MAX_IDENTIFIER_LENGTH] = {
```

Comment:
How about adding a comment like this:
// The order of module name MUST be the same with `mmon_module_id` starting from MMON_MODULE_ALL+1

Reply:
Good idea. Done.